the-blue-alliance-ios/View Controllers/Settings/SettingsTableViewController.swift

```diff
@@ -0,0 +1,200 @@
+            case 1:
+                urlString = "https://github.com/the-blue-alliance/the-blue-alliance-ios"
+                self.tableView.deselectRow(at: indexPath, animated: true)
+                
+            case 2:
+                // TODO: Make a contributors list on Github
+                self.tableView.deselectRow(at: indexPath, animated: true)
+                
+            case 3:
+                // TODO: Make a changelog
+                self.tableView.deselectRow(at: indexPath, animated: true)
+                
+            default:
+                fatalError("This row does not exist")
+            }
+            
+            if let urlString = urlString {
+                let url = URL(string: urlString)
+                if let url = url, UIApplication.shared.canOpenURL(url) {
+                    UIApplication.shared.open(url, options: [:], completionHandler: nil)
+                }
+            }
+            
+        case 3:
+            if indexPath.row == 0 {
+                let alertController = UIAlertController(title: "Delete network cache", message: "Are you sure you want to delete all the network cache data ? This action is irreversible.", preferredStyle: .alert)
+                
+                let confirmAction = UIAlertAction(title: "Delete", style: .destructive) { (deleteAction) in
+                    // TODO: Handle network cache deletion
+                }
+                
+                let cancelAction = UIAlertAction(title: "Cancel", style: .cancel) { (deleteAction) in
+                    self.tableView.deselectRow(at: indexPath, animated: true)
+                }
+                
+                alertController.addAction(confirmAction)
+                alertController.addAction(cancelAction)
+                
+                self.present(alertController, animated: true, completion: nil)
+            } else {
+                let alertController = UIAlertController(title: "Delete app data", message: "Are you sure you want to delete the app data ? This action is irreversible.", preferredStyle: .alert)
+                
+                let confirmAction = UIAlertAction(title: "Delete", style: .destructive) { (deleteAction) in
+                    // TODO: Handle app data deletion
+                }
+                
+                let cancelAction = UIAlertAction(title: "Cancel", style: .cancel) { (deleteAction) in
+                    self.tableView.deselectRow(at: indexPath, animated: true)
+                }
+                
+                alertController.addAction(confirmAction)
+                alertController.addAction(cancelAction)
+                
+                self.present(alertController, animated: true, completion: nil)
+            }
+            
+            
+        default:
+            fatalError("This section does not exist")
+        }
+    }
+    
+    override func tableView(_ tableView: UITableView, shouldHighlightRowAt indexPath: IndexPath) -> Bool {
```

Comment:
Let's not do this - just set the selection style on the table view cells you want to disable to none. Probably worth it to do it inside of the SwitchTableViewCell on setup